src/components/DropDown/DropDown.scss

```diff
@@ -0,0 +1,93 @@
+@import './../../styles/utils/vars';
+
+$borderWidth: 1px;
+$itemHeight: 32px;
+$itemVerticalPadding: 4px;
+$indentAbs4px: calc($itemHeight + 2 * $itemVerticalPadding + 4px);
+
+.dropDown {
+  cursor: pointer;
+  box-sizing: border-box; // ?
+  transition: all 0.2s ease-out;
+  border-color: #B4BDC4;
+  border-style: solid;
+  color: $colorGrayPrimary;
+  border-width: $borderWidth;
+  height: calc($itemHeight + 2 * $itemVerticalPadding);
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding-left: 12px;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 21px;
+
+  &::before {
+    position: absolute;
+    content: attr(data-dropdown-description);
+    /* Small text 12 */
+    font-family: inherit; // ?
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 15px;
+    color: $colorGraySecondary;
```

Comment:
Colors variables are not included into commit and they are not in main. So add file with variables too.